test-cases/longevity/longevity-gce-custom-d1-hybrid-raid.yaml

```diff
@@ -0,0 +1,36 @@
+test_duration: 900
+n_db_nodes: 6
+n_loaders: 2
+n_monitor_nodes: 1
+
+prepare_write_cmd: [
+  "latte run --duration 82286400 -P offset=0 --threads 10 --concurrency 128 --sampling 5s --function load scylla-qa-internal/custom_d1/workload1/latte/custom_d1_workload1.rn",
```

Comment:
It's actually the number of iterations
So it's the number of rows, and not actual time

The tool doesn't split it into two different arguments

Reply:
ugh, let's have some doc's about it as it's very confusing

Reply:
the docs of the tool itself say that:
```
    -d, --duration <TIME | COUNT>
            Number of cycles or duration of the main benchmark phase

            [default: 60s]
```

I don't know it it's worth duplicating into SCT, we'll see if we'll do much more with it, maybe it would need a bit more docs on SCT end.

Reply:
@fruch please add a doc/comment.

Reply:
add comment to explain that duration isn't really duration in the prepare commands